src/featurestorebundle/delta/DeltaDataHandler.py

```diff
@@ -0,0 +1,49 @@
+import datetime as dt
+from delta import DeltaTable
+from featurestorebundle.db.TableNames import TableNames
+from featurestorebundle.delta.DeltaMergeConfig import DeltaMergeConfig
+from featurestorebundle.entity.Entity import Entity
+from featurestorebundle.feature.FeatureList import FeatureList
+from featurestorebundle.feature.FeatureStore import FeatureStore
+from featurestorebundle.metadata.MetadataWriter import MetadataWriter
+from logging import Logger
+from pyspark.sql import SparkSession
+
+
+class DeltaDataHandler:
```

Comment:
Maybe all these Handlers shoudl be a subclass of an interface with a `write` function?